cpp/include/cudf/hashing/detail/murmur_hash64.cuh

```diff
@@ -0,0 +1,227 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <cudf/detail/utilities/algorithm.cuh>
+#include <cudf/detail/utilities/hash_functions.cuh>
+
+#include <thrust/pair.h>
+
+namespace cudf {
+namespace hashing {
+namespace detail {
+
+// MurmurHash3_x64_128 implementation from
+// https://github.com/aappleby/smhasher/blob/master/src/MurmurHash3.cpp
+//-----------------------------------------------------------------------------
+// MurmurHash3_64_128 was written by Austin Appleby, and is placed in the public
+// domain. The author hereby disclaims copyright to this source code.
+// Note - The x86 and x64 versions do _not_ produce the same results, as the
```

Comment:
is this true for GPU version as well?

Reply:
You mean the results statement? 
I would assume so since the code is nearly identical to the CPU version.